src/api/unified/symbol_manager.cpp

```diff
@@ -27,9 +27,9 @@ static const string LIB_AF_BKND_SUFFIX = ".dll";
 #else
 static const string LIB_AF_BKND_PREFIX = "libaf";
 #if defined(__APPLE__)
-static const string LIB_AF_BKND_SUFFIX = ".dylib";
+static const string LIB_AF_BKND_SUFFIX = ".3.dylib";
 #else
-static const string LIB_AF_BKND_SUFFIX = ".so";
+static const string LIB_AF_BKND_SUFFIX = ".so.3";
```

Comment:
@pavanky  May  be we can append the version number programmatically instead of hard coding it ?